docs/consensus_decisions.md

```diff
@@ -66,4 +66,4 @@ The group believes that the known value of this feature can be sufficiently cove
 **Discussion:**
 The cost analysis of the nested selectors feature was performed in the absence of sufficient in-field experience of use in production systems.
 In result, the group's decision to not currently incorporate the feature is based on the lack of sufficient known value that would require them, which the group recognizes may change in the future.
```

Comment:
Both this paragraph and the following one say "in result". This is also oddly worded. I'd suggest:

> Because the inclusion of nested selectors is not a blocker for the initial release and because the need for nested selectors remains subject to debate, ultimately the group decided not to incorporate the feature into this version of the standard.

Reply:
@aphillips Would you be open to having the current text be accepted provisionally in this PR, and then improving the language of Consensus 6 in a separate PR? I'm getting the sense that this one is a bit more divisive than the rest, and that it might be good to have a more focused discussion on its particulars.

Reply:
Go for it, although these suggestions are more about English prosody than content and I think we've about wrung any additional value out of this thread.